srcpkgs/adom/template

```diff
@@ -21,16 +21,16 @@ case "${XBPS_TARGET_MACHINE}" in
 		;;
 	x86_64)
 		distfiles="${_download_url}/adom_linux_debian_64_${version}.tar.gz"
-		checksum=588ddc3b680c8952fc1030747cca0047af64e48d1111eb014eaa094f44b6c9a5
+		checksum=b74a310e4180633d75ef26742933796261b475252a4123c6c492ee4ce96d48dd
 		;;
 	i686)
 		distfiles="${_download_url}/adom_linux_debian_32_${version}.tar.gz"
-		checksum=7deeaebb87be28f4e44c98384161783a6d93aa5dc4aa2a11559acf24a64dd934
+		checksum=20a900c425236205668db3fb052491d9c4fd3ff30765b1281802ff8a7cc374dd
 		;;
 	arm*)
 		_platform=arm
```

Comment:
I think this line can be removed, the variable isn’t used anywhere. Guess it was overlooked earlier. ;)

Reply:
good catch :)

Reply:
Well, I did the last update (a long time ago), so that one’s on me. :)